packages/components/src/components/post-header/post-header.scss

```diff
@@ -12,6 +13,9 @@
   --global-header-minimal-height: 24px;
   --main-header-height: 56px;
   --header-height: calc(var(--global-header-height) + var(--main-header-height));
+  @include utilities.high-contrast-mode() {
+    display: block;
```

Comment:
There would be an issue with the stickiness if you display the header as a block. Can you add the border to the local-header instead?